packages/mapviewer/src/modules/i18n/locales/de.json

```diff
@@ -88,7 +88,8 @@
     "cms_release_url": "https://www.geo.admin.ch/de/#Neue-Daten-und-Aktualisierungen",
     "commune": "Gemeinde",
     "compare": "Vergleichen",
-    "confirm_remove_all_features": "Wollen Sie wirklich alle Objekte löschen?",
+    "confirm_delete": "alles löschen",
+    "confirm_remove_all_features": "Möchten Sie wirklich Ihre gesamte Zeichnung löschen? Dieser Vorgang kann nicht rückgängig gemacht werden.",
```

Comment:
I'd maybe split this in two lines, and have a warning sign before the part that says "this can't be cancelled"